src/Sql/Sql/ServerActiveDirectoryAdministrator/Services/AzureSqlServerActiveDirectoryAdministratorAdapter.cs

```diff
@@ -40,30 +42,24 @@ public class AzureSqlServerActiveDirectoryAdministratorAdapter
         public IAzureContext Context { get; set; }
 
         /// <summary>
-        /// A private instance of ActiveDirectoryClient
+        /// A private instance of MicrosoftGraphClient
         /// </summary>
-        private ActiveDirectoryClient _activeDirectoryClient;
+        private MicrosoftGraphClient _microsoftGraphClient;
 
         /// <summary>
-        /// Gets or sets the Azure ActiveDirectoryClient instance
+        /// Gets or sets the Azure MicrosoftGraphClient instance
         /// </summary>
-        public ActiveDirectoryClient ActiveDirectoryClient
+        public MicrosoftGraphClient MicrosoftGraphClient
         {
             get
             {
-                if (_activeDirectoryClient == null)
+                if (_microsoftGraphClient == null)
                 {
-                    _activeDirectoryClient = new ActiveDirectoryClient(Context);
-                    if (!Context.Environment.IsEndpointSet(AzureEnvironment.Endpoint.Graph))
-                    {
-                        throw new ArgumentException(string.Format(Microsoft.Azure.Commands.Sql.Properties.Resources.InvalidGraphEndpoint));
-                    }
-                    _activeDirectoryClient = new ActiveDirectoryClient(Context);
+                    _microsoftGraphClient = AzureSession.Instance.ClientFactory.CreateArmClient<MicrosoftGraphClient>(Context, AzureEnvironment.Endpoint.Graph);
```

Comment:
Please set the tenant ID for the client:

`_microsoftGraphClient.TenantId = Context.Tenant.Id;` (need validation)